src/PlusWidgets/CMakeLists.txt

```diff
@@ -40,7 +40,9 @@ SET(${PROJECT_NAME}_INCLUDE_DIRS
   ${CMAKE_CURRENT_BINARY_DIR}
   ${CMAKE_CURRENT_SOURCE_DIR}
   )
-IF(CMAKE_VERSION VERSION_GREATER_EQUAL "3.8")
+# If CMake 3.7 is the minimum version then the next line can be

```

Comment:
VERSION_GREATER_EQUAL is introduced in CMake 3.7.
In the build instructions, CMake 3.7.2 version is described. I'm wondering if we should make it the minimum required CMake version in the top-level CMakeLists.txt.